.env.example

```
@@ -1,16 +1,16 @@
+# correspond to $SUPABASE_PUBLIC_URL
 PUBLIC_SUPABASE_URL=
+# correspond to $ANON_KEY
 PUBLIC_SUPABASE_ANON_KEY=
 
-PUBLIC_HOSTNAME=127.0.0.1:5173
-PUBLIC_URL=http://$PUBLIC_HOSTNAME
+ADMIN_URL=http://$PUBLIC_HOSTNAME
```

Vu qu'on enlève `PUBLIC_HOSTNAME` on peut juste mettre
```suggestion
ADMIN_URL=
```

---

Yes je le règle dans la PR #28